packages/ui/src/components/va-collapse/VaCollapse.vue

```diff
@@ -55,10 +55,11 @@
 <script lang="ts">
 import { computed, defineComponent, shallowRef } from 'vue'
 
-import { useKeyboardOnlyFocus, useColors, useSyncProp, useTextColor } from '../../composables'
+import { useKeyboardOnlyFocus, useColors, useSyncProp, useTextColor, useBem } from '../../composables'
 import { useAccordionItem } from '../va-accordion/hooks/useAccordion'
 
 import { generateUniqueId } from '../../services/utils'
+import pick from 'lodash/pick.js'
```

Comment:
Please move this import after the import from `'vue'`.
The basic idea is to group the external imports at the top of all imports.